tests/Microsoft.Bot.Builder.Dialogs.Adaptive.Tests/Tests/TestScriptTests/HttpRequestMock/httpluis.mock.dialog

```diff
@@ -0,0 +1,23 @@
+﻿{
+    "$schema": "../../../../tests.schema",
+    "$kind": "Microsoft.Test.HttpRequestSequenceMock",
+    "url": "https://westus.api.cognitive.microsoft.com/luis/prediction/v3.0/apps/00000000-0000-0000-0000-000000000000/*",
+    "content": "\"query\": \"overwrite\"",
```

Comment:
You can use a pattern here that is not sensitive to the app id, right?

Reply:
Yes. But I included an id here to give an example for how to match same query against different luis mocks (like top level and sub level would return different intents for same query)

Reply:
But this kind of my complaint.  LUIS is complicated to mock which is why getting a real response from LUIS and verifying it was the approach I took.  The case where you mock directly is much less likely.

---
In reply to: [471885860](https://github.com/microsoft/botbuilder-dotnet/pull/4453#discussion_r471885860) [](ancestors = 471885860)